Core/NoOpRunner.Core/NoOpRunner.cs

```diff
@@ -123,7 +123,8 @@ public void HandleKeyPress(KeyPress keyPress)
 
         private void InitializeGameState()
         {
-            GamePlatforms = new GamePlatforms(32, 32);
+            //Common aspect ration
+            GamePlatforms = new GamePlatforms(16*3, 9*3);
```

Comment:
Fix platform generation while you are at it - move the magic numbers into some constants, and use them both here and in `StaticShape` so that platforms are generated across the whole game screen